inc/namespace.php

```diff
@@ -590,3 +595,19 @@ function get_report_only_content_security_policies() : array {
 	 */
 	return apply_filters( 'altis.security.browser.report_only_content_security_policies', $policies );
 }
+
+/**
+ * Restrict CORS origin
+ * 
+ * @return mixed
+ */
+function restrict_cors_origin( $result ) : mixed {
```

Comment:
Should the return type be `bool | WP_Error` rather than `mixed`?

Reply:
Updated to make it `mixed | WP_Error` since this is being used for `rest_pre_dispatch` which expects `mixed`.

Reply:
I also removed return type in the function because union types does not seem supported pre PHP 8.0. It is set only in the doc block.